apturicovid/viewController/stats collection view/StatsFooterView.swift

```diff
@@ -1,33 +1,23 @@
 import UIKit
+import SafariServices
 
 //MARK: - StatsHeaderView
 class StatsFooterView: UICollectionReusableView {
 
     @IBOutlet weak var linkButton: AutoLayoutButton!
     
     @IBAction func linkTap(_ sender: Any) {
-        openUrlLink()
+        openURL()
     }
     
     static var identifier: String { String(describing: self) }
-    var linkUrl: URL?
+    var openURL: (() -> Void)!
     
     override func prepareForReuse() {
         linkButton.setTitle(nil, for: .normal)
-        linkUrl = nil
     }
     
-    func setup(with text: String, linkUrl: URL){
-        linkButton.setTitle(text, for: .normal)
-        self.linkUrl = linkUrl
-    }
-    
-    private func openUrlLink(){
-        
-        guard let url = linkUrl else { return }
-        
-        if UIApplication.shared.canOpenURL(url){
-            UIApplication.shared.open(url, options: [:], completionHandler: nil)
-        }
+    func setup(with text: String, _ openURL: @escaping () -> Void) {
+        self.openURL = openURL
```

Comment:
You deleted `linkButton.setTitle(text, for: .normal)` and now the label title isn't translated anymore